samples/attested_tls/common/cert_verifier.cpp

```diff
@@ -11,7 +11,15 @@
 #include <mbedtls/platform.h>
 #include <mbedtls/ssl.h>
 #include <string.h>
-#include "../../common/utility.h"
+#include "utility.h"
+
+#ifdef CLIENT_CERT_VERIFY_CALLBACK
+#define TLS_ENCLAVE TLS_CLIENT
+#endif
+
+#ifdef SERVER_CERT_VERIFY_CALLBACK
```

Comment:
Would it be better to use #elif given that `CLIENT_CERT_VERIFY_CALLBACK` and `SERVER_CERT_VERIFY_CALLBACK` should be mutually exclusive.

For example,
```
#if defined(CLIENT_CERT_VERIFY_CALLBACK)
...
#elif defined(SERVER_CERT_VERIFY_CALLBACK)
...
#else
// Maybe error out given that TLS_ENCLAVE must be defined.
#endif